java-api-core/src/com/kinvey/java/model/KinveyBatchInsertError.kt

```diff
@@ -0,0 +1,12 @@
+package com.kinvey.java.model
+
+import com.google.api.client.util.Key
+
+data class KinveyBatchInsertError (
```

Comment:
I'm wondering if we need to extends from `GenericJson`, I guess not right?

Reply:
Yes, I used extending from `GenericJson` for response classes in my last implementation of methods for multi-insert in `SyncManager`. Now it's none needed. Removed.